test.js

```diff
@@ -209,9 +210,11 @@ describe('easy stackup function', function() {
   // NOTE: (mc) Perhaps instead of these tests, we should check that we're passing
   // the correct things to gerber-to-svg, whats-that-gerber, and pcb-stackup-core?
   it('has deterministic top and bottom svgs if ids are given', function(done) {
+    var exampleGerber1 = fs.createReadStream('./integration/boards/arduino-uno/arduino-uno.gko')
+    var exampleGerber2 = fs.createReadStream('./integration/boards/arduino-uno/arduino-uno.plc')
```

Comment:
Readable streams are not reusable, so this won't work. Use `readFileSync` instead and the test passes.

As an aside, you'll wanna make sure you use `path.join(__dirname, ...)` to ensure the tests will run cross platform.